celery/app/trace.py

```diff
@@ -17,8 +16,6 @@
 from kombu.serialization import prepare_accept_content
 from kombu.utils.encoding import safe_repr, safe_str
 
-import celery
-import celery.loaders.app
```

Comment:
i wonder if these imports are what caused the issue, and if putting them behind a `if TYPE_CHECKING:` would fix it